### ISSUE TITLE
owtie-build: error while loading shared libraries: libtbbmalloc_proxy.so.2: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by ndaniel** [ndaniel/fusioncatcher#40]
In some cases Bowtie 1.2 would raise this error:
```
owtie-build: error while loading shared libraries: libtbbmalloc_proxy.so.2: cannot open shared object file: No such file or directory
```
on some older RedHat.

**Comment by ndaniel** [comment]
The solution was to re-compile bowtie on again against older glibc, and copy the executables to the tools subdirectory.

**Comment by ndaniel** [comment]
This has been fixed in v0.99.7c. If you have v0.99.7c and still have this issue then please re-install FusionCatcher again and this will solve it!
